### PR TITLE
Put some delay in-between disabling and enabling universal SSL

### PR DESCRIPTION
**Pull request by stefanor**
It seems to be asynchronous on their side.

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: pycloudflare/models.py [hunk 1]
```diff
@@ -202,9 +203,20 @@ def re_verify(self):
...
         """
-        for state in (False, True):
-            self._service.update_ssl_universal_settings(
-                self.id, {'enabled': state})
+        self._service.update_ssl_universal_settings(
+            self.id, {'enabled': False})
+        # This is asynchronous on CF's side. So probe until it seems to have
+        # been deprovisioned, and then wait a little longer to be safe.
+        for i in range(5):
+            sleep(1)
```
Comment (RayeN) [inline pycloudflare/models.py]: 😿

**Comment by RayeN** [comment]
Changelog?

File: pycloudflare/models.py (hunk 1 above)
Comment (bearnard) [inline pycloudflare/models.py]: 😿

**Pull request merged**